src/portal/src/app/project/list-project/list-project.component.ts

```diff
@@ -208,6 +208,8 @@ export class ListProjectComponent implements OnDestroy {
                     this.clrLoad(st);
                     this.statisticHandler.refresh();
                 }
+            }, error => {
+                this.msgHandler.handleError(error);
```

Comment:
The information about deleting the project is displayed in the local event. If you do this here, what about other error messages displayed in the local event? This needs to be discussed

Reply:
Keep both